pkg/client/didconfig/didconfig.go

```diff
@@ -66,10 +67,14 @@ func WithJSONLDDocumentLoader(documentLoader jsonld.DocumentLoader) Option {
 	}
 }
 
+type didResolver interface {
+	Resolve(did string, opts ...vdrapi.DIDMethodOption) (*did.DocResolution, error)
+}
+
 // WithVDRegistry defines a vdr service.
-func WithVDRegistry(vdrRegistry vdrapi.Registry) Option {
+func WithVDRegistry(didResolver didResolver) Option {
```

Comment:
Should it still be called `WithVDRegistry`?

Reply:
`WithDIDResolver` would be a better name, but the consumers of this will need to update. @sandrask do you know where this is used?